30_differential_abundance.Rmd

```diff
@@ -339,26 +340,29 @@ that we specify.
 
 
 ```{r ancombc2, warning = FALSE, eval=TRUE}
-
 # Agglomerate data to genus level and add this new abundance table to the altExp slot
-altExp(tse, "genus") <- mergeFeaturesByRank(tse, "genus")
+altExp(tse, "genus") <- agglomerateByRank(tse,
+                                          rank = "genus")
 
 # Identify prevalent genera
-prevalent.genera <- getPrevalentFeatures(altExp(tse, "genus"), detection = 0, prevalence = 30/100)
+prevalent.genera <- getPrevalentFeatures(altExp(tse, "genus"),
+                                         detection = 0,
+                                         prevalence = 30 / 100)
 
-# Run ANCOM-BC at the genus level and only including the prevalent genera
+# Load package
+library(ANCOMBC)
 
-out <- ancombc2(
-  data = altExp(tse, "genus")[prevalent.genera, ],
-  assay_name = "counts", 
-  fix_formula = "Geographical_location", 
-  p_adj_method = "fdr", 
-  prv_cut = 0,
-  group = "Geographical_location", 
-  struc_zero = TRUE, 
-  neg_lb = TRUE,
-  global = TRUE # multi group comparison will be deactivated automatically 
-)
+# Run ANCOM-BC at the genus level and only including the prevalent genera
+out <- ancombc2(data = altExp(tse, "genus")[prevalent.genera, ],
+                assay_name = "counts",
```

Comment:
check if `assay.type` works as an argument name with the latest ANCOMBC version from Github?

Reply:
Apparently `assay.type` is not yet supported by ANCOMBC (GitHub version).

Reply:
Yeah. Stilll in the making.